recipes/obspy/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+## echo "print('Testing an example package.')" > $SP_DIR/example.py
+cp -r $RECIPE_DIR/.. $SRC_DIR
```

Comment:
What does this do?  Why is it in build.sh and not bld.bat?

If you don't actually need it, then you can get rid of bld.bat and build.sh, and instead have a

build:
    script: python setup.py install

entry in meta.yaml.